cmd/protoc-gen-crd/main.go

```diff
@@ -74,15 +75,30 @@ func generate(request *plugin.CodeGeneratorRequest) (*plugin.CodeGeneratorRespon
 
 	m := protomodel.NewModel(request, false)
 
-	filesToGen := make(map[*protomodel.FileDescriptor]bool)
+	legacyChannelFilesToGen := make(map[*protomodel.FileDescriptor]bool)
```

Comment:
nit: this can be optimized by changing the map to a `map[*protomodel.FileDescriptor]struct{}` so that you don't have to look at the bools later (and avoid allocating an extra bit for it, but that's not consequential)